blob/service.go

```diff
@@ -55,10 +57,18 @@ func NewService(
 // Allows sending multiple Blobs atomically synchronously.
 // Uses default wallet registered on the Node.
 // Handles gas estimation and fee calculation.
-func (s *Service) Submit(ctx context.Context, blobs []*Blob) (uint64, error) {
+func (s *Service) Submit(ctx context.Context, blobs []*Blob, options ...Option) (uint64, error) {
 	log.Debugw("submitting blobs", "amount", len(blobs))
 
-	resp, err := s.blobSumitter.SubmitPayForBlob(ctx, types.OneInt().Neg(), 0, blobs)
+	s.feeParams = &feeParams{
```

Comment:
it'd be nice and idiomatic to add a `DefaultParams() &feeParams` or similar in the options.go, then we can also add a test around the defaults